src/txParsers.ts

```diff
@@ -269,14 +269,16 @@ export const parseSignedTx = (unparsedSignedTx: unknown): SignedTransaction => {
 }
 
 export const parseRawTx = (unparsedUnsignedTx: unknown): RawTransaction => {
-    const [body, nativeScriptWitnesses, auxiliaryData] = parseTuple(
+    const [body, item1, item2] = parseTuple(
         unparsedUnsignedTx,
         ParseErrorReason.INVALID_RAW_TX_CBOR,
         parseTxBody,
         dontParse,
         dontParse,
     )
-
-    return {body, nativeScriptWitnesses, auxiliaryData}
+    // older versions of cardano-cli did not include nativeScriptWitnesses
+    if (item2 === undefined) {
+        return {body, nativeScriptWitnesses: [], auxiliaryData: item1}
+    }
+    return {body, nativeScriptWitnesses: item1, auxiliaryData: item2}
```

Comment:
@KubqoA even though we concluded that this is not necessary since newer version of cardano-cli always include all three items, I found out that ~half of our tests in hw-cli use this format. So we have three options:
1. re-create all the old tests with new hw-cli
2. support the old format here (I prefer this)
3. add the supporting code to hw-cli test preparation function (I see no benefit over 2.)

cc @janmazak

Reply:
I don't see any harm in supporting the old format here. This is likely to change with Plutus, anyway, and we don't know how right now (nor does IOHK).